src/api/search.ts

```diff
@@ -239,7 +239,7 @@ const getProductSearch = async ({
     window.adobeDataLayer.push((dl: any) => {
       dl.push({
         event: 'search-results-view',
-        eventInfo: { ...dl.getState(), searchUnitId: SEARCH_UNIT_ID },
+        eventInfo: { ...dl.getState(), searchUnitId: SEARCH_UNIT_ID, searchResults: results?.data?.productSearch },
```

Comment:
Hey @miakobchuk ,
the event you need to track for PLP view is the one above `category-results-view`.
And when I look in the event-info I already see the search results data on it. 
What are you adding here as `searchResults` ?
![Screenshot 2024-08-26 at 11 21 15](https://github.com/user-attachments/assets/9fc24367-c988-436a-9ea5-b6ea966c5dcc)

Reply:
I am using productView from results as it contains some additional attributes, including: price, rating, season, model...

Reply:
searchRequest data was replaced with specific attributes data